languages/ocaml.scm

```diff
@@ -630,7 +630,7 @@
   ":" @append_spaced_softline
 )
 (module_binding
-  ":" @append_spaced_softline
+  ":" @append_input_softline
```

Comment:
This means we're not opinionated on this. Is there any way we could format using our customary single/multi line logic instead?

Reply:
We could be opinionated, but I think this warrants another PR. Also, if I'm not mistaken it relates to #217